Mathlib/Analysis/Calculus/AddTorsor/AffineMap.lean

```diff
@@ -30,4 +28,23 @@ theorem contDiff {n : WithTop ℕ∞} (f : V →ᴬ[𝕜] W) : ContDiff 𝕜 n f
   apply f.contLinear.contDiff.add
   exact contDiff_const
 
+theorem differentiable (f : V →ᴬ[𝕜] W) : Differentiable 𝕜 f :=
+  f.contDiff.differentiable le_rfl
+
+theorem differentiableAt (f : V →ᴬ[𝕜] W) {x : V} : DifferentiableAt 𝕜 f x :=
+  f.differentiable x
+
+theorem differentiableOn (f : V →ᴬ[𝕜] W) {s : Set V} : DifferentiableOn 𝕜 f s :=
+  f.differentiable.differentiableOn
+
+theorem differentiableWithinAt (f : V →ᴬ[𝕜] W) {s : Set V} {x : V} :
+    DifferentiableWithinAt 𝕜 f s x :=
+  f.differentiableAt.differentiableWithinAt
+
+@[simp] theorem fderiv (f : V →ᴬ[𝕜] W) {x : V} : fderiv 𝕜 f x = f.contLinear := by
+  conv_lhs => rw [f.decomp]
+  rw [fderiv_add f.contLinear.differentiableAt]; swap
+  · exact differentiableAt_const _
+  simp only [fderiv_const, Pi.zero_apply, add_zero, ContinuousLinearMap.fderiv]
```

Comment:
There is something wrong here. Either we need `f.decomp` to not be about `Function.const` or we need to add
```
@[simp, fun_prop]
lemma differentiableAt_Function_const{𝕜 : Type*} [NontriviallyNormedField 𝕜] 
  {E : Type*} [NormedAddCommGroup E] [NormedSpace 𝕜 E] 
  {F : Type*} [NormedAddCommGroup F] [NormedSpace 𝕜 F] {x : E} (c : F) :
  DifferentiableAt 𝕜 (Function.const _ c) x := differentiableAt_const _
```
(and presumably many other similarly duplicated lemmas about `Function.const`). 

In both cases we should have `rw [fderiv_add (by fun_prop) (by fun_prop)]`.

Reply:
Will you do it in this PR then or do you want to merge this PR and come back to this issue?

Reply:
I won't do it in this PR, it will be another 400 lines of boring mechanical things, let's not confuse it with something slightly more interesting.